### ISSUE TITLE
Need to bring Bing maps as a layer using Mapfish

### ISSUE THREAD

**Issue by chrismeteu** [mapfish/mapfish-print#511]
Even after upgrading to Mapfish version 2.1.2 i see no way to include Bing maps as a layer. Is there any plans to add them as an option or possibly a work around to get them to work for a printed PDF?

**Comment by DafyddLlyr** [comment]
I'd be interested in this as well. Are there any plans to add support for Bing Maps or any suggested workarounds?

**Comment by sbrunner** [comment]
No, we didn't have any plan to implement it, but if a pull request is provided we will take time to review and merge it :-)
